src/components/navbar/Navbar.module.scss

```diff
@@ -0,0 +1,105 @@
+@import '../../utils/variables';
+$tiny: 348px;
+
+.navContainer {
+    box-shadow: 0px 2px 6px rgba(0, 0, 0, 0.3);
+    position: sticky;
+    top: 0;
+    z-index: 20;
+
+    * {
+        box-sizing: border-box;
+    }
+    .topNavbar,
+    .bottomNavbar {
+        height: 75px;
+        padding: 10px;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+    .linkContainer {
+        height: 100%;
+        width: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: flex-end;
+
+        @media (max-width: $medium) {
+            display: none;
+        }
+    }
+
+    .topNavbar {
+        background-color: $topNavBg;
+        @media (max-width: $medium) {
+            display: none;
+        }
+    }
+    .bottomNavbar {
+        background-color: white;
+        position: relative;
+
+        @media (max-width: $tiny) {
+            flex-direction: column;
+            height: auto;
+        }
+
+        .logoContainer {
+            display: flex;
+            margin: 0 10px;
+            height: 100%;
+            min-width: 250px;
+
+            @media (max-width: $tiny) {
+                min-width: auto;
+                width: 100%;
+                height: auto;
+            }
+        }
+        .navBurgerContainer {
+            cursor: pointer;
+            margin: 10px 10px;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            min-height: 40px;
+            min-width: 40px;
+            display: none;
+
+            @media (max-width: $medium) {
+                display: inherit;
+            }
+            @media (max-width: $tiny) {
+                margin-bottom: 0;
+            }
+
+            .burger {
+                position: relative;
+                width: 90%;
+                height: 3px;
+                background: $blue;
+                pointer-events: none;
+
+                &::before {
+                    content: '';
+                    width: 100%;
+                    height: 100%;
+                    position: absolute;
+                    top: -12px;
+                    left: 0;
+                    background: inherit;
+                }
+                &::after {
+                    content: '';
+                    width: 100%;
+                    height: 100%;
+                    position: absolute;
+                    bottom: -12px;
+                    left: 0;
+                    background: inherit;
```

Comment:
I'm unsure what this is doing but the contents of these are the same so you could join them together to use the same styles by doing `&::before, &::after {`

Reply:
This is the dropdown menu burger icon, the difference would be the top and bottom. I didn't want to contain them because I would have to make 3 selector calls. If you think it would be better I could do something else like.
`&::before,
&::after {
    content: '';
    width: 100%;
    height: 100%;
    position: absolute;
    background: inherit;
    left: 0;
}
&::before {
    top: -12px;
}
&::after {
    bottom: -12px;
}`

It is about the same lines, It just seems more confusing to me.

Reply:
the chat code block didn't turn out as pretty as I thought.